definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -1693,6 +1693,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "testDNPronounced",
+    "UserRole": "caseworker-divorce-courtadmin_betacitizen",
```

Comment:
caseworker-divorce-courtadmin_betacitizen?? what is this role for?

Reply:
I was testing you. You passed, too. :)